src/Stratis.Bitcoin/Configuration/Logging/EventId.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Stratis.Bitcoin.Configuration.Logging
```

Comment:
This code is commented out, should this be removed?

Reply:
Removed.